<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>231</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CloudFuse - Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="group_account">
     <property name="title">
      <string>Account</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item row="1" column="0">
         <widget class="QLabel" name="label_email">
          <property name="text">
           <string>Email:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="input_email"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_password">
          <property name="text">
           <string>Password:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="input_password">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="group_name">
     <property name="title">
      <string>Name</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_nameFormat">
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="input_name"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_result">
        <property name="text">
         <string>Result:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_example">
        <property name="text">
         <string>example</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>radio_anon</tabstop>
  <tabstop>radio_account</tabstop>
  <tabstop>input_email</tabstop>
  <tabstop>input_password</tabstop>
  <tabstop>input_name</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>393</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>393</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_account</sender>
   <signal>toggled(bool)</signal>
   <receiver>button_authenticate</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_anon</sender>
   <signal>toggled(bool)</signal>
   <receiver>radio_dontcopy</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>73</x>
     <y>357</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_account</sender>
   <signal>toggled(bool)</signal>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
